server/src/FormMetaData/formmetadata.entity.ts

```diff
@@ -19,4 +20,7 @@ export class FormMetaData extends CustomBaseEntity {
 
   @Column({ type: 'varchar', length: '100', nullable: false })
   chefsFormId: string;
+
```

Comment:
Could we also add the versionID for the forms? `data.version.id`
And also the schema for the version, which is `data.version.schema`

Reply:
Sure thing, added them to FormData.